CVLByExample/summary/multi-contract/certora/specs/spec_with_multicontract.spec

```diff
@@ -0,0 +1,15 @@
+using B as b;
+methods {
+    function b.foo(uint256) external returns(uint256) envfree; // foo of B
+    function foo(uint) external returns(uint256) envfree; // foo of the currentContract A
+}
+
+rule checkMultiContractWithEnvFree() {
```

Comment:
demonstares how to call multi contract from spec

Reply:
I will split this directory to 3
1. mult-contract with this spec (out of the summary direstory because no summarization here).
2. function summary
3. internal external

Reply:
remove this spec
add from index - how to call external contract from spec: CVLByExample/LiquidityPool/certora/specs/pool_link.spec